mswindows/osgeo4w/python-grass.bat.tmpl

```diff
@@ -0,0 +1,18 @@
+@echo off
+rem #########################################################################
+rem #
+rem # GRASS initialization bat script for Python environment (OSGeo4W)
+rem #
+rem #########################################################################
+
+rem
+rem Set environmental variables
+rem
+call "%~dp0\o4w_env.bat"

```

Comment:
in python-qgis.bat, there is also

`call py3_env.bat`

as there is still also python2 living inside OSGeo4W. maybe we should get rid of python 2 at some point in OSGeo4W. python 2 is still default:

```
run o-help for a list of available commands
C:\>python
Python 2.7.14 (v2.7.14:84471935ed, Sep 16 2017, 20:25:58) [MSC v.1500 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> exit()
```

or am I overlooking something.

Reply:
Yes, but python-grass.bat calls GRASS `env.bat` file where `py3_env.bat` is introduced already.

Reply:
ok, keep it there.